nebula-logger/core/main/log-management/flexipages/LogRecordPage.flexipage-meta.xml

```diff
@@ -525,6 +525,96 @@
         <name>Facet-12145474-d012-4654-ba48-4100dbbbc893</name>
         <type>Facet</type>
     </flexiPageRegions>
+    <flexiPageRegions>
+        <itemInstances>
+            <fieldInstance>
+                <fieldInstanceProperties>
+                    <name>uiBehavior</name>
+                    <value>readonly</value>
+                </fieldInstanceProperties>
+                <fieldItem>Record.AsyncContextType__c</fieldItem>
+                <identifier>RecordAsyncContextType_cField</identifier>
+            </fieldInstance>
+        </itemInstances>
+        <name>Facet-dd5e3466-ea03-4853-be00-33411ed46ec6</name>
+        <type>Facet</type>
+    </flexiPageRegions>
+    <flexiPageRegions>
+        <itemInstances>
+            <fieldInstance>
+                <fieldInstanceProperties>
+                    <name>uiBehavior</name>
+                    <value>readonly</value>
+                </fieldInstanceProperties>
+                <fieldItem>Record.AsyncContextParentJobId__c</fieldItem>
+                <identifier>RecordAsyncContextParentJobId_cField</identifier>
+                <visibilityRule>
+                    <criteria>
+                        <leftValue>{!Record.AsyncContextParentJobId__c}</leftValue>
+                        <operator>NE</operator>
+                    </criteria>
+                </visibilityRule>
+            </fieldInstance>
+        </itemInstances>
+        <itemInstances>
+            <fieldInstance>
+                <fieldInstanceProperties>
+                    <name>uiBehavior</name>
+                    <value>readonly</value>
+                </fieldInstanceProperties>
+                <fieldItem>Record.AsyncContextChildJobId__c</fieldItem>
+                <identifier>RecordAsyncContextChildJobId_cField</identifier>
+                <visibilityRule>
+                    <criteria>
+                        <leftValue>{!Record.AsyncContextChildJobId__c}</leftValue>
+                        <operator>NE</operator>
+                    </criteria>
+                </visibilityRule>
+            </fieldInstance>
+        </itemInstances>
+        <itemInstances>
+            <fieldInstance>
+                <fieldInstanceProperties>
+                    <name>uiBehavior</name>
+                    <value>readonly</value>
+                </fieldInstanceProperties>
+                <fieldItem>Record.AsyncContextTriggerId__c</fieldItem>
+                <identifier>RecordAsyncContextTriggerId_cField</identifier>
+                <visibilityRule>
+                    <criteria>
+                        <leftValue>{!Record.AsyncContextChildJobId__c}</leftValue>
```

Comment:
shouldn't this be `{!Record.AsyncContextChildJobId__c}` ?

Reply:
Yeah, great catch! I'll fix this